lib/DDGC/DB/ResultSet/InstantAnswer.pm

```diff
@@ -4,6 +4,22 @@ package DDGC::DB::ResultSet::InstantAnswer;
 use Moose;
 extends 'DDGC::DB::Base::ResultSet';
 use namespace::autoclean;
+use DateTime;
+use DateTime::Format::Pg;
 
-1;
+sub last_modified {
+    my ( $self ) = @_;
+    my $last_modified =  $self->search( {}, {
+        columns => [
+            { last_modified => { max => 'updated' } },
+        ],
+    } )->one_row->get_column('last_modified');
+
+    return DateTime::Format::Pg->parse_datetime(
+        $last_modified
+    ) if $last_modified;
 
+    return DateTime->new( year => 1970 );
```

Comment:
If the query plan has a MAX( updated ) we return a DateTime instance of that, else epoch. Example query for spice repo:

```sql
SELECT MAX( "updated" ) 
  FROM "instant_answer" "me" 
WHERE ( "dev_milestone" = 'live' OR "dev_milestone" = 'development' OR "dev_milestone" = 'testing' OR "dev_milestone" = 'complete' ) AND "repo" = 'spice' 
  LIMIT '1'
```

`LIMIT 1` is superfluous, but is added by `->one_row` which returns a Result class instance we can `->get_column` on.